sdk/src/azure/iot/az_iot_provisioning_client.c

```diff
@@ -564,22 +595,27 @@ AZ_NODISCARD az_result az_iot_provisioning_client_parse_received_topic_and_paylo
   return AZ_OK;
 }
 
-AZ_NODISCARD az_result az_iot_provisioning_client_get_request_payload(
+AZ_NODISCARD az_iot_provisioning_client_payload_options
+az_iot_provisioning_client_payload_options_default()
+{
+  return (az_iot_provisioning_client_payload_options){ 0 };
+}
```

Comment:
nit: other iot files initialize each element explicitly. consider making this similar

https://github.com/Azure/azure-sdk-for-c/blob/399f1dbd09c2208f044cef4a4b4b97d115817de3/sdk/src/azure/iot/az_iot_hub_client.c#L29-L36